docs/Community/Repositories.md

```diff
@@ -0,0 +1,122 @@
+      bot automation for pull requests
+      unless otherwise approved by the [CGC](governance/README.md).
+   4. Must follow other [contributing guidelines](Contributing.md)
+      unless otherwise approved by the [CGC](governance/README.md).
+   5. Must live under `github.com/openenclave/<project-name>`.
+   6. Code repositories must be approved by
+      [SIG-Architecture](sig-architecture/README.md).
+      Non-code repositories must be approved by the
+      [CGC](governance/README.md).
+
+### Temporary forks
+
+Sometimes the Open Enclave project will want changes to other external
+repositories.  Ideally, such changes would be contributed
+directly to the upstream project.  However, in some cases it may be
+necessary to use a temporary fork for some period of time, whether for
+development, or for use while the upstream submission is being reviewed
+and iterated on.  In such cases, a fork might be done under an individual's
+github organization or the openenenclave organization.  (A fork is not
+needed at all if a branch can be created in the related project.)
+
+If the Open Enclave SDK needs to directly depend on a fork of the related
+project, it is generally better for the fork to be done under the openenclave
+organization.  This makes it clear that Open Enclave itself, rather than
+a just a random developer, depends on the fork.
+
+#### Rules
+
+   1. Must adopt the Open Enclave [Code of Conduct](Governance.md#code-of-conduct)
+      statement in their repo.
+   2. Must use the same license and copyright rules as the forked project.
+   3. Must adopt the [DCO](Contributing.md#developer-certificate-of-origin)
+      bot automation for pull requests
+      unless otherwise approved by the [CGC](governance/README.md).
+   4. Must follow other [contributing guidelines](Contributing.md)
+      unless otherwise approved by the [CGC](governance/README.md).
+   5. Must live under `github.com/openenclave/<project-name>`.
+   6. Repository must be approved by [SIG-Architecture](sig-architecture/README.md).
+
+Other than rule 2, all other rules are the same as for core repositories.
+
+## Creating Repositories
+
+Requests for creating repositories can be made by filing an
+[Open Enclave issue](https://github.com/openenclave/openenclave/issues)
+if one does not already exist.
+
+In addition to stating any deviations from the Open Enclave defaults
+discussed under Rules above, any request to create a new repository
+should also answer the following questions:
+
+1. Should github issues be filed in the new repo, or in the main
+   openenclave repo?
+2. Should issues in the new repo be discussed in the master openenclave
+   triage meeting, or in some separate process?
+3. Should one or more of the existing SIGs (e.g.,
+   [SIG-Architecture](sig-architecture/README.md),
+   [SIG-Testing](sig-testing/README.md)) oversee relevant aspects of the
+   new repo, or not?
+4. Will the repo be public or private, and if private, why?
```

Comment:
I would suggest that this document only deals with public repos since we don't really discuss what might be reasonable criteria for asking for a private repo to be created. 

The one standard use case that we have for that is security advisories, and that will be addressed specifically under the documentation for that process.

Reply:
Done